model/src/gx_outp.F90

```diff
@@ -1115,9 +1115,9 @@ SUBROUTINE GXEXPO
                TAUWNX, TAUWNY, XWI, DIA, LLWS, IX, IY, LAMBDA )
           CALL W3SPR4 (A, CG, WN, EMEAN, FMEAN,  FMEAN1,      &
                WNMEAN, AMAX, UABS, UDIRR,               &
-#ifdef W3_FLX5
+# ifdef W3_FLX5
                TAUA, TAUADIR, RHOAIR,             &
```

Comment:
@MathieuDutSik , same for lines: 1118, 1120?

Reply:
I am for adding the space because it makes the code clearer.
Imagine programming without indentation, this is much harder. Indentation is here to help.

Reply:
It is subjective that this makes the code cleaner.   The code needs to be consistent, the current standard is no space, and we will persist with this.  Apologies this was not explicit in the WW3 code standards, I can add it there if necessary to make it cleaner.  In the meantime, these spaces need to be reviewed for this PR to be accepted.

Reply:
Hi @MathieuDutSik, thank you for understanding the collective nature.  The Standards are somewhat new and came about during the related discussions on `#ifdef` removal.  They are here, [WW3 Code Standards](https://github.com/NOAA-EMC/WW3/wiki/WW3-Code-Style-Standards), in the wiki.

Reply:
> Even if it's allowed, let's not add the space.

Why?

Reply:
@aronroland  For consistency, we've explicitly added this to the WW3 code standards that no space will be between the # and the if or end statements.  Apologies this was not included before.  These additional spaces will need to be removed before this PR can move forward.  This is great work and we're looking forward to continued improvements to WW3!

Reply:
As it was not included before at this time in the standards the PR should have been accepted and the white spaces should be corrected in another PR or the standards could have been expanded ...